### ISSUE TITLE
Return proper validator status for GRPC queries

### ISSUE THREAD

**Issue by alpe** [confio/tgrade#124]
tldr; 
Set Validator status in https://github.com/confio/tgrade/blob/v0.6.0-rc2/x/poe/contract/tgrade_valset.go#L243 to "bounded" when it is an [`active_validator`](https://github.com/confio/poe-contracts/blob/v0.6.0/contracts/tgrade-valset/src/msg.rs#L379) otherwise `unbonded`.

## Background:
The status in the SDK can be:
* Bonded
* Unbonding
* Unbonded

## Update:
I have looked more into the details and this needs some better refinement to integrate with PoE:
Following the status definitions from the [sdk doc](https://docs.cosmos.network/v0.42/modules/staking/01_state.html#validator)
I propose a simplified version with 2 states only:
* Unbonded: validator is not in the active valset
* Bonded: validator is in the active valset

We don't have automatic `Unbonding` as there are no delegator's stake to protect. Moving out of the active valset can currently happen when:
* Jailed
* Validator's stake is reduced
* Validator's EP are reduced
* Other validators have a higher combined Stake  EP value
* Number of total validators is reduced

Unless there are some concrete use cases that I miss, let's make things simpler
